### PR TITLE
feat(docs): tutorial on nginx

### PR DESCRIPTION
**Pull request by amyshwang**

### CODE REVIEW

File: blog/2022-08-05-setting-basic-auth-nginx.md [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+author_url: https://kovidrathee.medium.com/
+author_image_url: https://miro.medium.com/fit/c/96/96/0*_CwYR2OmNap47tQO.jpg
+description:
+  How to implement Nginx Basic Auth for QuestDB.
+keywords:
+  - nginx
+  - data security
+  - timeseries
+
+image: /img/blog/2022-08-05/banner.png
```
[the remark this answers is not in the document]
Comment (amyshwang) [reply blog/2022-08-05-setting-basic-auth-nginx.md:14]: This is okay.

File: blog/2022-08-05-setting-basic-auth-nginx.md [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+title: Setting up Basic Auth for QuestDB using Nginx
+author: Kovid Rathee
+author_title: Guest post
+author_url: https://kovidrathee.medium.com/
+author_image_url: https://miro.medium.com/fit/c/96/96/0*_CwYR2OmNap47tQO.jpg
+description:
+  How to implement Nginx Basic Auth for QuestDB.
+keywords:
+  - nginx
+  - data security
```
Comment (amyshwang) [inline blog/2022-08-05-setting-basic-auth-nginx.md:11]:
```suggestion
  - security
  - auth
  - proxy
```

File: blog/2022-08-05-setting-basic-auth-nginx.md [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+  - timeseries
+
+image: /img/blog/2022-08-05/banner.png
+tags: [tutorial, nginx, data security]
```
Comment (amyshwang) [inline blog/2022-08-05-setting-basic-auth-nginx.md:15]:
```suggestion
tags: [tutorial, nginx, security, auth, proxy]
```

**Review by amyshwang — COMMENTED** [review]
From Nic.

**Review by argshook — APPROVED** [review]

**Review by tris0laris — APPROVED** [review]

**Pull request merged**